### ISSUE TITLE
Support magic arguments

### ISSUE THREAD

**Issue by bollwyvl** [bollwyvl/hy_kernel#12]
Neither cell or line magics can accept parameters: they result in parse errors.

**Comment by actsasgeek** [comment]
Is there a way to get my matplotlib inline whilst this issue is open?

```
%matplotlib inline
Using matplotlib backend: MacOSX
Magics with parameters are not supported
```

**Comment by bollwyvl** [comment]
Not yet. The magics are not real python, so hy can't make ast for them. I
tried masking them in comments, but that didn't work either, as hy strips
comments.

The hy to ast to code approach is brittle as hell anyway. The real answer
will be to somehow wrap magics:

(%matplotlib inline)

As well as somehow handle cell magics that don't expect code. PRs or
suggestions welcome!

**Comment by bollwyvl** [comment]
Actually, just fixed this with #17, now on pypi as 0.3.0.

It breaks the code up around magics and compiles them as discrete chunks.

Still won't do inline magic!

**Comment by actsasgeek** [comment]
sweet, just tried it out. Thanks!
